multidim-interop/src/generator.ts

```diff
@@ -118,6 +118,7 @@ function buildSpec(containerImages: { [key: string]: () => string }, { name, dia
                     transport,
                     is_dialer: true,
                     ip: "0.0.0.0",
+                    redis_addr: "redis:6379",
```

Comment:
If I recall correctly this is the default value? Either way, I prefer this to happen in a separate pull request.

Reply:
That is true, it is the default value. But `ip: "0.0.0.0"` is also the default value. To keep it consistent...
But I like your opinion to put this in a different PR. I'll remove it here.